src/client/reducers/appReducer.ts

```diff
@@ -25,19 +35,21 @@ export interface AppState {
  * including user/channel and booleans based on user/channel types,
  * as well as commonly referenced app URLs
  */
-export const initialState: AppState = {
-  apiURL: sd.API_URL,
-  appURL: sd.APP_URL,
-  artsyURL: sd.ARTSY_URL,
-  channel: sd.CURRENT_CHANNEL,
-  forceURL: sd.FORCE_URL,
-  isAdmin: sd.USER.type === "Admin",
-  isEditorial: sd.CURRENT_CHANNEL.type === "editorial",
-  isPartnerChannel: sd.CURRENT_CHANNEL.type === "partner",
-  metaphysicsURL: sd.GRAPHQL_ENDPOINT,
-  user: sd.USER,
-}
+export const getInitialState = () =>
+  ({
```

Comment:
Changed this to a function to help with tests -- this ensures that sd values are always fresh when `appReducer` is called